card_db/en_us/translation.txt

```diff
@@ -0,0 +1,71 @@
+# Translation Instructions
```

Comment:
I went back and forth on this.  The pain of upkeep vs having the information for a translator all in one directory.  Even the more mature translations still have cards that need translation, so that eliminated just having it in the xx directory.

So it can be moved if we think translators will find it.  Or more likely, remembered if the files are to be sent to someone to translate via email.  It is only documentation, so it can easily be moved.

To be honest, the best place would be to have it in the json files themselves, at the top in comments.  It would take preprocessing the files.  But I understand that is frowned upon.